prac_05/word_occurences.py

```diff
@@ -0,0 +1,17 @@
+"""
+Word Occurences
+Estimate: 15 minutes
+Actual: 11 minutes
+"""
+
+word_to_count = {}
+user_string = input("Text: ").lower().split()
+for word in user_string:
+    try:
+        word_to_count[word] = word_to_count[word] + 1
+    except KeyError:
+        word_to_count[word] = 1
+print(word_to_count)
```

Comment:
Seems this is an artefact from testing, consider removing

Reply:
I believe you are right, whoops, thank you.